TS type error in 8.2.0+

## 🐛 Bug Report

Using the lib with TS immediately fails with a type error.

## To Reproduce

```
node_modules/i18next-parser/dist/index.d.ts:86:36 - error TS2552: Cannot find name 'RegEx'. Did you mean 'RegExp'?

86   keepRemoved?: boolean | readonly RegEx[];
                                      ~~~~~

  node_modules/typescript/lib/lib.es5.d.ts:1064:13
    1064 declare var RegExp: RegExpConstructor;
                     ~~~~~~
    'RegExp' is declared here.
```

Introduced here: https://github.com/i18next/i18next-parser/commit/217635a8e63725a74f081d8697ae16d0657014c4

## Expected behavior

No type error.

## Your Environment

- *runtime version*: node v18.12.0
- *i18next version*: 8.3.0
- *os*: Windows 10
- *TS version*: 5.1.6

--- a/index.d.ts
+++ b/index.d.ts
@@ -94,7 +94,7 @@ export interface UserConfig {
   defaultNamespace?: string;
   defaultValue?: string | ((locale?: string, namespace?: string, key?: string) => string);
   indentation?: number;
-  keepRemoved?: boolean | readonly RegEx[];
+  keepRemoved?: boolean | readonly RegExp[];
   keySeparator?: string | false;
   lexers?: {
     hbs?: (SupportedLexer | CustomLexer | LexerConfig)[];

